aps/lib/infinity_aps/cfg/server.ex

```diff
@@ -72,7 +54,7 @@ defmodule InfinityAPS.Configuration.Server do
          :ok <- File.write(file, config_data) do
       # Temporary way to apply configuration
       Logger.info("Configuration saved, rebooting")
-      Nerves.Runtime.reboot()
+      Runtime.reboot()
```

Comment:
@David-Gil I'm sorry, you were correct about the reason for the test failure earlier. The

```
warning: function Nerves.Runtime.reboot/0 is undefined (module Nerves.Runtime is not available) lib/infinity_aps/cfg/server.ex:56
```

is indeed failing the build because warnings in the aps project are treated as errors (cfg wasn't configured that way). If you remove the reboot() line here and the `alias Nerves.Runtime` earlier in this file, the CI should pass and we can re-introduce it later (or better, avoid having to reboot to apply the config changes)